ioctl/cmd/action/stake2create.go

```diff
@@ -0,0 +1,100 @@
+// Copyright (c) 2020 IoTeX Foundation
+// This is an alpha (internal) release and is not suitable for production. This source code is provided 'as is' and no
+// warranties are given as to title or non-infringement, merchantability or fitness for purpose and, to the extent
+// permitted by law, all liability for your use of the code is disclaimed. This source code is governed by Apache
+// License 2.0 that can be found in the LICENSE file.
+
+package action
+
+import (
+	"encoding/hex"
+
+	"github.com/spf13/cobra"
+
+	"github.com/iotexproject/iotex-core/action"
+	"github.com/iotexproject/iotex-core/ioctl/config"
+	"github.com/iotexproject/iotex-core/ioctl/output"
+)
+
+// Multi-language support
+var (
+	stake2CreateCmdUses = map[config.Language]string{
+		config.English: "create AMOUNT_IOTX CANDIDATE_NAME STAKE_DURATION [DATA] [--auto-restake]",
+		config.Chinese: "create IOTX数量 候选人姓名 权益持续时间 [数据] [--auto-restake]",
+	}
+
+	stake2CreateCmdShorts = map[config.Language]string{
+		config.English: "create stake on IoTeX blockchain",
+		config.Chinese: "在区块链上创建质押",
+	}
+
+	stake2FalgAutoRestakeUsages = map[config.Language]string{
+		config.English: "auto restake without power decay",
+		config.Chinese: "自动质押，权重不会衰减",
+	}
+)
+
+// stake2CreateCmd represents the stake2 create command
+var stake2CreateCmd = &cobra.Command{Use: config.TranslateInLang(stake2CreateCmdUses, config.UILanguage),
+	Short: config.TranslateInLang(stake2CreateCmdShorts, config.UILanguage),
+	Args:  cobra.RangeArgs(3, 4),
+	RunE: func(cmd *cobra.Command, args []string) error {
+		cmd.SilenceUsage = true
+		err := create(args)
+		return output.PrintError(err)
+	},
+}
+
+func init() {
+	registerWriteCommand(stake2CreateCmd)
+	stake2CreateCmd.Flags().BoolVar(&stake2AutoRestake, "auto-restake", false,
+		config.TranslateInLang(stake2FalgAutoRestakeUsages, config.UILanguage))
+}
+
+func stake2Create(args []string) error {
+	var amount = args[0]
+	var candidateName = args[1]
```

Comment:
we also need to validate the name, will let u know the detail method soon

Reply:
refer to staking/validations.go

func IsValidCandidateName(s string) bool

Reply:
> refer to staking/validations.go
> 
> func IsValidCandidateName(s string) bool

ok, I will change ioctl/validator/validator.go:func ValidateCandidateName() to this func.